packages/storybook/src/nimble/table/table.mdx

```diff
@@ -9,9 +10,23 @@ The <Tag name={tableTag}/> is a component that offers a way to render tabular da
 For information about configuring table columns, see **Table Column Configuration** and **Table Column Types**.
```

Comment:
> For information about configuring table columns, see **Table Column Configuration** and **Table Column Types**.

Can we link the referenced docs? Also, we no longer have a page called "Table Column Types", so maybe we just remove that.